spaceshooter/spaceShooter.py

```diff
@@ -494,7 +494,7 @@ def update(self):
     ## now as we delete the mob element when we hit one with a bullet, we need to respawn them again
     ## as there will be no mob_elements left out 
     for hit in hits:
-        score += 50 - hit.radius         ## give different scores for hitting big and small metoers
+        score += 50 - hit.radius -int(score/3000 )       ## give different scores for hitting big and small metoers
```

Comment:
Can it be that an expression `50 - hit.radius -int(score/3000 )` gives a negative value?

Plus: is it unconvinient to add whitespace inside brackets as well as no space around minus sign.